Cython/Utility/ModuleSetupCode.c

```diff
@@ -1042,6 +1056,23 @@ static CYTHON_INLINE PyObject * __Pyx_PyDict_GetItemStrWithError(PyObject *dict,
   #define __Pyx_SET_SIZE(obj, size) Py_SIZE(obj) = (size)
 #endif
 
+#if CYTHON_AVOID_THREAD_UNSAFE_BORROWED_REFS && PY_VERSION_HEX >= 0x030d00b1
+#define __Pyx_PyList_GetItemRef(o, i) PyList_GetItemRef(o, i)
+static CYTHON_INLINE int __Pyx_PyDict_GetItemRef(PyObject *dict, PyObject *key, PyObject **result) {
+  return PyDict_GetItemRef(dict, key, result);
+}
```

Comment:
Looks like these could both be macros.
```suggestion
#define __Pyx_PyList_GetItemRef(o, i)  PyList_GetItemRef(o, i)
#define __Pyx_PyDict_GetItemRef(dict, key, result)  PyDict_GetItemRef(dict, key, result)
```